libraries/Microsoft.Bot.Builder/TokenResolver.cs

```diff
@@ -0,0 +1,134 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Linq;
+using System.Security.Claims;
+using System.Security.Principal;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Bot.Schema;
+using Newtonsoft.Json.Linq;
+
+namespace Microsoft.Bot.Builder
+{
+    internal class TokenResolver
+    {
+        private const int PollingInterval = 1000;   // Poll for token every 1 second.
```

Comment:
PollingIntervalMs

Reply:
I suggest using TimeSpan, more readable. Most methods receive timeouts as TimeSpan or int, so we can use that to our advantage as well. Note below how you don't need the comment because the code itself is self-explanatory

` private const TimeSpan PollingInterval = TimeSpan.FromSeconds(1);`

Then you can pass that TimeSpan to whatever time out, or you can get the milliseconds using the TotalMilliseconds property (IIRC)

Reply:
Went with Carlos's suggestion